.circleci/orbs/helix-post-deploy/orb.yml

```diff
@@ -84,13 +92,15 @@ commands:
               if [ "<< parameters.newrelic_url >>" != "" ]; then
                 url="<< parameters.newrelic_url >>"
               else
+                actionNS = "<< parameters.action_namespace >>"
+                actionPackage = "<< parameters.action_package >>"
```

Comment:
Do you really need those extra variables ?

Reply:
No, but I found the code more legible this way :) 
The placeholders get quite long...